src/components/SchemaViewer/index.stories.js

```diff
@@ -171,37 +172,18 @@ export const demo = () => (
 );
 
 export const customThemes = () => {
-  const darkTheme = createMuiTheme({
-    palette: {
-      type: 'dark',
-    },
-  });
-  const colorfulTheme = createMuiTheme({
-    palette: {
-      background: {
-        paper: '#000',
-      },
-      text: {
-        primary: '#ffc107',
-        secondary: '#ffc53d',
-        hint: '#ddd',
-      },
-      divider: '#4f4f4f',
-    },
-  });
-
   return (
     <Fragment>
-      <h3>Dark Theme</h3>
-      <ThemeProvider theme={darkTheme}>
+      <h3>Taskcluster Dark Theme</h3>
+      <ThemeProvider theme={TASKCLUSTER_THEME.darkTheme}>
         <SchemaViewer schema={objectSchemas.complexObjectExample} />
       </ThemeProvider>
-      <h3>Colorful Theme ex1</h3>
-      <ThemeProvider theme={colorfulTheme}>
+      <h3>Taskcluster Light Theme</h3>
+      <ThemeProvider theme={TASKCLUSTER_THEME.lightTheme}>
         <SchemaViewer schema={demoSchemas.workerList} />
       </ThemeProvider>
-      <h3>Colorful Theme ex2</h3>
-      <ThemeProvider theme={colorfulTheme}>
+      <h3>Taskcluster Dark Theme</h3>

```

Comment:
Is there a reason for having two Taskcluster Dark Themes?

Reply:
Also, there seems to be a white border under "Show Source".

<img width="1363" alt="Screen Shot 2020-02-25 at 5 09 30 PM" src="https://user-images.githubusercontent.com/3766511/75292150-a4181d00-57f1-11ea-9125-2f1e8a3b55b0.png">

It would be nice to not have this gap white but rather the same background color as the rest.

Reply:
Applied changes to remove example and white gap.